src/js/dataMapping/search/awardTableColumns.jsx

```diff
@@ -338,9 +345,10 @@ const tabsWithCovidCols = [
 if (kGlobalConstants.CARES_ACT_RELEASED) {
     // Insert columns for COVID-19
     tabsWithCovidCols.forEach((tab) => {
-        tab.splice(5, 0, covidDefCCol);
-        tab.splice(6, 0, covidObligationsCol);
-        tab.splice(7, 0, covidOutlaysCol);
+        tab.splice(5, 0, descriptionCol);
```

Comment:
this isn't. actually a tab with covid data. Can we do this another way? Maybe we could just rename the var on line `347` to reflect that we're doing this data mapping strategy for more columns than just covid ones?